front_end/package.json

```diff
@@ -93,7 +96,10 @@
   },
   "scripts": {
     "test": "karma start --reporters dots --single-run --browsers Firefox && mocha ./node_tests/**/*.test.js",
+    "build:css": "node-sass ./src/main.scss -o dist --importer build_utils/sass-importer.js --source-map dist/main.css.map",
     "build:dist": "gulp build:dist",
-    "start": "gulp serve"
+    "build:static": "cp -r src/static/* dist",
+    "build:fonts": "cp -r node_modules/font-awesome/fonts dist",
+    "start": "mkdir -p dist && npm run build:static && npm run build:fonts && npm run build:css && npm run build:css -- -w & gulp serve"
```

Comment:
you actually need to run it twice because the watch task from node-sass doesn't do an initial build :(